CMakeLists.txt

```diff
@@ -3,11 +3,15 @@ if(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
 endif(COMMAND cmake_policy)
 
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.10")
+  cmake_policy(SET CMP0015 NEW)
+endif()
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13")
+  cmake_policy(SET CMP0081 NEW)
+endif()
+
 set(CMAKE_BUILD_TYPE "Release" CACHE STRING
   "one of: Release, Debug, RelWithDebInfo or MinSizeRel")
-if (${CMAKE_MAJOR_VERSION} GREATER 2 OR ${CMAKE_MINOR_VERSION} GREATER 7)
-  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Release;Debug;RelWithDebInfo;MinSizeRel")
```

Comment:
Why is this line removed ?

Reply:
I removed the `if` test since the minimum required CMake version is 2.8 so any CMake that makes it this far would meet the conditions of the if test.

I removed the `set_property` since I don't think that it is needed for current versions of CMake.  The only difference that I can see in the CMakeCache.txt with and without this line is an entry in the "INTERNAL Cache Entries" portion of the file.  The `CMAKE_BUILD_TYPE` variable up in the "EXTERNAL Cache Entries" portion of the file is the same with or without the `set_property`

Reply:
Ok thanks for the explanation